rts/Lua/LuaHandleSynced.h

```diff
@@ -60,9 +60,7 @@ class CSyncedLuaHandle : public CLuaHandle
 		bool AllowCommand(const CUnit* unit, const Command& cmd, int playerNum, bool fromSynced, bool fromLua) override;
 
 		std::pair <bool, bool> AllowUnitCreation(const UnitDef* unitDef, const CUnit* builder, const BuildInfo* buildInfo) override;
-		bool AllowUnitTransfer(const CUnit* unit, int newTeam, int reason) override;
-		// @deprecated Use the reason-based signature instead. This signature is subject to change in future versions.
-		bool AllowUnitTransfer(const CUnit* unit, int newTeam, bool capture);
```

Comment:
delete this version of AllowUnitTransfer. Keep the capture one but deprecate it.